inc/notificationtargetcommonitilobject.class.php

```diff
@@ -989,15 +989,28 @@ function getDataForObject(CommonDBTM $item, array $options, $simple = false) {
          $data["##$objettype.assigntogroups##"] = implode(', ', $groups);
       }
 
-      $data["##$objettype.solution.type##"]='';
-      if ($item->getField('solutiontypes_id')) {
-         $data["##$objettype.solution.type##"]
-                              = Dropdown::getDropdownName('glpi_solutiontypes',
-                                                          $item->getField('solutiontypes_id'));
-      }
+      $data["##$objettype.solution.type##"] = '';
+      $data["##$objettype.solution.description##"] = '';
+
+      $itilsolution = new ITILSolution();
+      $solution = $itilsolution->getFromDBByCrit([
+         'itemtype'  => $objettype,
+         'items_id'  => $item->fields['id'],
+         'status'    => CommonITILValidation::ACCEPTED
```

Comment:
I'm not sure about ACCEPTED.

In fact, for the notification, we want the last submitted solution